README.md

```diff
@@ -63,9 +63,11 @@ each project representing a specific task and usually a [Microservice](http://en
 
 In order to use the endpoints for geospatial operations, the following environment variables need to be set:
 
-* `PG_DATABASE` : name of the PostgreSQL database
-* `PG_USER`: PostgreSQL user (needs to have relevant permissions on tables to be queried)
-* `PG_PASSWORD`: Password for PostgreSQL user
+* `PG_HOST`: host running PostgreSQL. Defaults to `localhost`
```

Comment:
May want to use the official Postgres envvars, which are basically what we have, minus the `_` underscore.